### PR TITLE
[MB-8067] Load testing SC & TOO Move Details orders, shipments, and service item APIs

### PR DESCRIPTION
**Pull request by duncan-truss**
## Description

This PR is part of the load testing the GHC API work for additional queries for orders, shipments, and service items on the move details page of SC and TOO users.  The PR to add the tasks to the load testing tool is here https://github.com/transcom/milmove_load_testing/pull/71.

Changes are required in the mymove repo to address the move queue queries returning the order id instead of the move id in the payload response.  This was something that stuck around because the initial implementation relied on the orders id more than the move id at the time.  Now the move locator is standardized in our URLs so we can replace the order id in the queue payloads now.

The other change I noticed was that the ListMTOServiceItems endpoint was accessible from the Services Counselor user, which should not be the case.  I added a basic authorization check to return forbidden for Services Counselor because the move will never have any service items until after the TOO approves the move/shipments.

## Reviewer Notes

If I have extra time I'll try to add more fidelity to the seed file when it comes to shipments and service items in our generated moves.

## Setup

Add any steps or code to run in this section to help others prepare to run your code:

```sh
make db_dev_e2e_populate
make server_run
make office_client_run
```

Verifying the move id is now in the payload instead of the orders id:
1. Login as the TOO or SC user
2. Inspect the data returned from the queue for a single move and note the id value (in the react-query developer tools or in the chrome dev inspector)
3. Check in your local database whether this id matches what is in the database for that move.

Verifying the services counselor can not make requests for service items:
1. Launch Postman with your GHC API collection and turn on the cookie interceptor
2. Login to the office app as a Services Counselor user
3. Return to postman and make an API request to the ListMTOServiceItems endpoint at `move_task_orders/:moveId/mto_service_items`

## Code Review Verification Steps

* [ ] If the change is risky, it has been tested in experimental before merging.
* [ ] Code follows the guidelines for [Logging](https://github.com/transcom/mymove/wiki/Backend-Programming-Guide#logging)
* [ ] The requirements listed in [Querying the Database Safely](https://github.com/transcom/mymove/wiki/Backend-Programming-Guide#querying-the-database-safely) have been satisfied.
* Any new migrations/schema changes:
  * [ ] Follow our guidelines for zero-downtime deploys (see [Zero-Downtime Deploys](https://github.com/transcom/mymove/wiki/migrate-the-database#zero-downtime-migrations))
  * [ ] Have been communicated to #g-database
  * [ ] Secure migrations have been tested following the instructions in our [docs](https://github.com/transcom/mymove/wiki/migrate-the-database#secure-migrations)
* [ ] There are no aXe warnings for UI.
* [ ] This works in [Supported Browsers and their phone views](https://github.com/transcom/mymove/tree/master/docs/adr/0016-Browser-Support.md) (Chrome, Firefox, IE, Edge).
* [ ] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
* [ ] User facing changes have been reviewed by design.
* [ ] Request review from a member of a different team.
* [ ] Have the Jira acceptance criteria been met for this change?

## References

* [Jira story](https://dp3.atlassian.net/browse/MB-8067) for this change

## Screenshots

If this PR makes visible UI changes, an image of the finished UI can help reviewers and casual
observers understand the context of the changes. A before image is optional and
can be included at the submitter's discretion.

Consider using an animated image to show an entire workflow instead of using multiple images. You may want to use GIPHY CAPTURE for this! 📸

_Please frame screenshots to show enough useful context but also highlight the affected regions._

### CODE REVIEW

File: pkg/handlers/ghcapi/mto_service_items.go [hunk 1]
```diff
@@ -117,7 +119,11 @@ type ListMTOServiceItemsHandler struct {
 
 // Handle handler that lists mto service items for the move task order
 func (h ListMTOServiceItemsHandler) Handle(params mtoserviceitemop.ListMTOServiceItemsParams) middleware.Responder {
-	logger := h.LoggerFromRequest(params.HTTPRequest)
+	session, logger := h.SessionAndLoggerFromRequest(params.HTTPRequest)
+
+	if !session.IsOfficeUser() || (!session.Roles.HasRole(roles.RoleTypeTOO) && !session.Roles.HasRole(roles.RoleTypeTIO)) {
```
Comment (jenniferlynparsons) [inline pkg/handlers/ghcapi/mto_service_items.go:124]: Please log an error and add an assertion in the test.

**Review by jenniferlynparsons — CHANGES_REQUESTED** [review]

**Review by Ryan-Koch — APPROVED** [review]
LGTM, though I did see that comment from JP about adding in a logging statement for the role check. That'd probably be a nice add on.

File: pkg/handlers/ghcapi/mto_service_items.go (hunk 1 above)
Comment (duncan-truss) [reply pkg/handlers/ghcapi/mto_service_items.go:124]: Added here https://github.com/transcom/mymove/pull/6669/commits/a936576ed056425e3e8c20a05a0bca5b531112e8

**Review by jenniferlynparsons — APPROVED** [review]
🎉 Looks good!

**Review by jenniferlynparsons — APPROVED** [review]
Ready to go!

**Comment by duncan-truss** [comment]
> Ready to go!

Thanks! I abandoned it when CircleCI stopped working on Friday.

**Pull request merged**